autosklearn/ensembles/multiobjective_dummy_ensemble.py

```diff
@@ -0,0 +1,248 @@
+from __future__ import annotations
+
+from typing import Any, Sequence
+
+import warnings
+
+import numpy as np
+
+from autosklearn.automl_common.common.utils.backend import Backend
+from autosklearn.constants import TASK_TYPES
+from autosklearn.data.validation import SUPPORTED_FEAT_TYPES
+from autosklearn.ensemble_building.run import Run
+from autosklearn.ensembles.abstract_ensemble import (
+    AbstractEnsemble,
+    AbstractMultiObjectiveEnsemble,
+)
+from autosklearn.ensembles.singlebest_ensemble import SingleModelEnsemble
+from autosklearn.metrics import Scorer, calculate_losses
+from autosklearn.pipeline.base import BasePipeline
+
+
+class MultiObjectiveDummyEnsemble(AbstractMultiObjectiveEnsemble):
+    def __init__(
+        self,
+        task_type: int,
+        metrics: Sequence[Scorer] | Scorer,
+        random_state: int | np.random.RandomState | None,
+        backend: Backend,
+    ) -> None:
+        """A dummy implementation of a multi-objective ensemble.
+
+        Builds ensembles that are individual models on the Pareto front each.
+
+        Parameters
+        ----------
+        task_type: int
+            An identifier indicating which task is being performed.
+
+        metrics: Sequence[Scorer] | Scorer
+            The metrics used to evaluate the models.
+
+        random_state: Optional[int | RandomState] = None
+            Not used.
+
+        backend : Backend
+            Gives access to the backend of Auto-sklearn. Not used.
+        """
+        self.task_type = task_type
+        if isinstance(metrics, Sequence):
+            if len(metrics) == 1:
+                warnings.warn(
+                    "Passed only a single metric to a multi-objective ensemble. "
+                    "Please use a single-objective ensemble in such cases."
+                )
+            self.metrics = metrics
+        else:
+            self.metric = [metrics]
```

Comment:
I would leave it as a warning as this would not stop Auto-sklearn from working. Also, this would only happen if the user willfully passes the `MultiObjectiveDummyEnsemble` to Auto-sklearn. If the user passes `[metric]` to Auto-sklearn without specifying the ensemble model, we would still default to `EnsembleSelection`.